components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -198,10 +198,14 @@ export class Panel extends React.Component<Props, State> {
     }
   }
 
-  openRewardsAddFundsPage () {
+  openRewardsAddFundsPage (id?: string) {
     chrome.tabs.create({
       url: 'chrome://rewards/#add-funds'
     })
+
+    if (id) {
```

Comment:
Feels unexpected to delete the notification in this function (and the parameter name doesn't really tell you how `id` is going to be used - maybe `notificationId` if we need to keep it), but could just be I don't understand the flow.

Reply:
renamed to `notificationId`. We need to do this in this function as with clicking on a button we open new tab and that is final step for the notification so it needs to be removed